### PR TITLE
drying up code for buttons

### PR DESCRIPTION
**Pull request by julien51**

### CODE REVIEW

File: unlock-app/src/components/creator/FatalError.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ export const WrongNetwork = ({ currentNetwork, requiredNetwork }) => (
     illustration="/static/images/illustrations/network.svg"
   >
     <p>
-      You’re currently on the
+      You’re currently on the
```
Comment (benwerd) [inline unlock-app/src/components/creator/FatalError.js]: Worth seeing if your editor can automatically trim whitespace from the end of lines. It's not a big deal but it'd be nice to keep the source clean.

File: unlock-app/src/components/interface/buttons/Button.js [hunk 2]
```diff
@@ -8,63 +8,59 @@ const clickAction = (e, action) => {
...
+  children,
+  backgroundColor,
+  backgroundHoverColor,
+  fillColor,
+  fillHoverColor,
+  ...props
+}) => {
+  return (
+    <BaseButton
+      backgroundColor={backgroundColor || 'var(--grey)'}
```
Comment (benwerd) [inline unlock-app/src/components/interface/buttons/Button.js]: Do we need this if `defaultProps` is being set below?

File: unlock-app/src/components/interface/buttons/layout/About.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import Svg from '../../svg'
 import { LayoutButton } from '../Button'
 
 const About = props => (
-  <LayoutButton href="/about" title="About" {...props}>
+  <LayoutButton href="/about" label="About" {...props}>
```
Comment (benwerd) [inline unlock-app/src/components/interface/buttons/layout/About.js]: Makes sense to use label here on the layout buttons - but let's continue to use `title` for UI elements?

File: unlock-app/src/components/creator/FatalError.js (hunk 1 above)
Comment (julien51) [reply unlock-app/src/components/creator/FatalError.js]: Meh. this is weird. I thought prettify would take care of that?

File: unlock-app/src/components/interface/buttons/Button.js (hunk 2 above)
Comment (julien51) [reply unlock-app/src/components/interface/buttons/Button.js]: I think you're right. Let me remove that.

File: unlock-app/src/components/interface/buttons/layout/About.js (hunk 3 above)
Comment (julien51) [reply unlock-app/src/components/interface/buttons/layout/About.js]: ok!

**Pull request merged**